src/blockencodings.h

```diff
@@ -74,14 +74,14 @@ struct PrefilledTransaction {
     SERIALIZE_METHODS(PrefilledTransaction, obj) { READWRITE(COMPACTSIZE(obj.index), Using<TransactionCompression>(obj.tx)); }
 };
 
-typedef enum ReadStatus_t
+using ReadStatus = enum ReadStatus_t
```

Comment:
In commit "refactor: use "using" over "typedef"" (4f510b061229f54b5c0d5dcbd91ea28cfca63dae)

Can just say `enum ReadStatus {` here. `ReadStatus_t` is never referenced.